### PR TITLE
Landing page design for Cloudlibz

### PR DESCRIPTION
**Pull request by kmehant**
selection_037
selection_038
selection_039
selection_040

### CODE REVIEW

**Comment by rehrumesh** [comment]
Hi @kmehant , 

Great work. Few improvements I want:

1. Remove the twitter/facebook icons and add gitter icon
2. Cloudlibz has a logo. use that logo.
3. nodecloud/goclouds are products, listing cloudlibs also in a green box makes it feels like it is also a product. so remove cloudlibz green box.
4. Remove the fun facts section, replace it with an image of current cloud provides like AWS, Google Cloud, Azure, Rackspace, Openshift, Alibaba cloud logos. Section title should be "Solutions towards multi-cloud providers"
5. Subscribe section -> Community. Should have links to email, github org link, and Gitter link
6. Remove meet the team section. Team will grow later.
7. Our Portfolio -> Projects

**Comment by rehrumesh** [comment]
I also noticed that you have committed node_modules directory. Remove those files and add 'node_modules" to gitignore

**Comment by kmehant** [comment]
Addressed the changes!

**Comment by rehrumesh** [comment]
Very good. Now there are few more improvements. You may have noticed that the square block in the logo does not match at all with the dark color. So inverting the colors would be fixing this.

Remove the middle box with gocloud and nodecloud logos. that section doesn't make sense.

Community links would look nice if they are in a row rather than a list.

File: index.html [hunk 1]
```diff
@@ -1,15 +1,258 @@
...
-  Author: ProBootstrap.com
-  Author URL: https://probootstrap.com
-  License: Released for free under the Creative Commons Attribution 3.0 license (probootstrap.com/license)
+<!DOCTYPE HTML>
+<!--
+	Aesthetic by gettemplates.co
+	Twitter: http://twitter.com/gettemplateco
+	URL: http://gettemplates.co
 -->
+<<<<<<< HEAD
```
Comment (rehrumesh) [inline index.html]: remove this

File: index.html [hunk 2]
```diff
@@ -1,15 +1,258 @@
...
+	<!-- Magnific Popup -->
+	<script src="js/jquery.magnific-popup.min.js"></script>
+	<script src="js/magnific-popup-options.js"></script>
+	<!-- Main -->
+	<script src="js/main.js"></script>
+
+	</body>
+</html>
+
+=======
```
Comment (rehrumesh) [inline index.html]: remove this as well.

File: index.html [hunk 3]
```diff
@@ -63,3 +306,4 @@ <h4>Stay tuned for more updates</h4>
 </body>
 
 </html>
+>>>>>>> parent of 2cc7ac4... Add google verification
```
Comment (rehrumesh) [inline index.html]: remove this as well.

**Pull request closed without merging**